packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -647,7 +662,7 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       }
       if (this.props.parentRef || this.chartContainer) {
         const container = this.props.parentRef ? this.props.parentRef : this.chartContainer;
-        const currentContainerWidth = container.getBoundingClientRect().width;
+        const currentContainerWidth = Math.max(container.getBoundingClientRect().width, this._calculateChartMinWidth());
```

Comment:
This call happens in the second render cycle, right?

Reply:
Yes